devolutions-gateway/openapi/ts-angular-client/api/jrec.service.ts

```diff
@@ -88,6 +90,88 @@ export class JrecService {
         return httpParams;
     }
 
+    /**
+     * Mass-deletes recordings stored on this instance
+     * If you try to delete more than 1,000,000 recordings at once, you should split the list into multiple requests to avoid timing out during the processing of the request.  The request processing consist in 1) checking if one of the recording is active, 2) counting the number of recordings not found on this instance.  When a recording is not found on this instance, a counter is incremented. This number is returned as part of the response. You may use this information to detect anomalies on your side. For instance, this suggests the list of recordings on your side is out of date, and you may want re-index.
```

Comment:
[nit] Something gone wrong with formatting here

Reply:
The generator is removing newlines, but I think it’s fine though.